sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,36 @@
+# 2021 SIG network annual report
+
+This report reflects back on CY 2020 and was written in April 2021.
+
+> Note: Each response should have supporting documentation, linked KEPs, graphs, data
+> Note: Err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+# Operational
+
+- How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+  - Is your [README](./README.md) accurate? 
+  - Have a CONTRIBUTING.md file?
+  - All subprojects correctly mapped and listed in [sigs.yaml](https://github.com/kubernetes/community/blob/master/sig-list.md)?
```

Comment:
Missing the network plumbing WG - @dcbw is this still active?

Reply:
@caseydavenport It's not technically an appproved Kube working group... we just started using the same Zoom 3+ years ago and never moved elsewhere. We can discuss starting that process though?

Reply:
@dcbw probably makes sense to make it official :grin: